src/mrinufft/io/nsp.py

```diff
@@ -29,14 +31,16 @@ def write_gradients(
     grad_filename: str,
     img_size: tuple[int, ...],
     FOV: tuple[float, ...],
-    in_out: bool = True,
+    TE_pos: float = 0.5,
```

Comment:
Some trajectories have multiple echoes per readouts.
Also since `TE_pos` is only used to check initial conditions, I would suggest to just use a boolean (with a more explicit name than `in_out` indeed)